OpenRA.Mods.Common/Traits/BotModules/PowerDownBotMoudle.cs

```diff
@@ -0,0 +1,148 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2019 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Traits
+{
+	[Desc("Manages AI powerdown.")]
+	public class PowerDownBotMoudleInfo : ConditionalTraitInfo
+	{
+		[Desc("Delay (in ticks) between toggle")]
```

Comment:
`Delay (in ticks) between toggling powerdown.`

Reply:
Thanks

Reply:
fixed